build/configs/esp32_DevKitC/kernel_tc/defconfig

```diff
@@ -0,0 +1,839 @@
+# CONFIG_MTD_SMART_SECTOR_ERASE_DEBUG is not set
+# CONFIG_MTD_SMART_ALLOC_DEBUG is not set
+# CONFIG_MTD_W25 is not set
+
+#
+# System Logging
+#
+# CONFIG_SYSLOG is not set
+# CONFIG_SYSLOG_TIMESTAMP is not set
+
+#
+# Database
+#
+# CONFIG_ARASTORAGE is not set
+
+#
+# Memory Management
+#
+# CONFIG_REALLOC_DISABLE_NEIGHBOR_EXTENSION is not set
+# CONFIG_MM_SMALL is not set
+CONFIG_MM_REGIONS=1
+CONFIG_MM_NHEAPS=1
+# CONFIG_GRAN is not set
+
+#
+# Power Management
+#
+# CONFIG_PM is not set
+
+#
+# Debug Options
+#
+CONFIG_DEBUG=y
+CONFIG_DEBUG_ERROR=y
+# CONFIG_DEBUG_WARN is not set
+CONFIG_DEBUG_VERBOSE=y
+
+#
+# Subsystem Debug Options
+#
+# CONFIG_DEBUG_LIB is not set
+# CONFIG_DEBUG_FS is not set
+# CONFIG_DEBUG_MM is not set
+# CONFIG_DEBUG_SCHED is not set
+# CONFIG_DEBUG_TASH is not set
+
+#
+# OS Function Debug Options
+#
+# CONFIG_ARCH_HAVE_HEAPCHECK is not set
+CONFIG_DEBUG_MM_HEAPINFO=y
+# CONFIG_DEBUG_IRQ is not set
+
+#
+# Driver Debug Options
+#
+# CONFIG_DEBUG_I2S is not set
+# CONFIG_DEBUG_SPI is not set
+
+#
+# System Debug Options
+#
+# CONFIG_DEBUG_SYSTEM is not set
+
+#
+# Stack Debug Options
+#
+CONFIG_ARCH_HAVE_STACKCHECK=y
+# CONFIG_STACK_COLORATION is not set
+
+#
+# Build Debug Options
+#
+CONFIG_DEBUG_SYMBOLS=y
+# CONFIG_FRAME_POINTER is not set
+
+#
+# Logger Module
+#
+# CONFIG_LOGM is not set
+
+#
+# Built-in Libraries
+#
+
+#
+# Standard C Library Options
+#
+CONFIG_STDIO_BUFFER_SIZE=64
+CONFIG_STDIO_LINEBUFFER=y
+CONFIG_NUNGET_CHARS=2
+CONFIG_LIB_HOMEDIR="/"
+CONFIG_LIBM=y
+# CONFIG_NOPRINTF_FIELDWIDTH is not set
+# CONFIG_LIBC_FLOATINGPOINT is not set
+CONFIG_LIBC_SCANSET=y
+# CONFIG_NOPRINTF_LONGLONG_TO_ASCII is not set
+# CONFIG_LIBC_IOCTL_VARIADIC is not set
+# CONFIG_LIBC_WCHAR is not set
+# CONFIG_LIBC_LOCALE is not set
+CONFIG_LIB_RAND_ORDER=1
+# CONFIG_EOL_IS_CR is not set
+# CONFIG_EOL_IS_LF is not set
+# CONFIG_EOL_IS_BOTH_CRLF is not set
+CONFIG_EOL_IS_EITHER_CRLF=y
+# CONFIG_LIBC_STRERROR is not set
+# CONFIG_LIBC_PERROR_STDOUT is not set
+CONFIG_LIBC_TMPDIR="/tmp"
+CONFIG_LIBC_MAX_TMPFILE=32
+CONFIG_ARCH_LOWPUTC=y
+# CONFIG_LIBC_LOCALTIME is not set
+# CONFIG_TIME_EXTENDED is not set
+CONFIG_LIB_SENDFILE_BUFSIZE=512
+# CONFIG_ARCH_OPTIMIZED_FUNCTIONS is not set
+
+#
+# Non-standard Library Support
+#
+
+#
+# Basic CXX Support
+#
+# CONFIG_C99_BOOL8 is not set
+# CONFIG_HAVE_CXX is not set
+
+#
+# External Libraries
+#
+# CONFIG_AVS_DEVICE_SDK is not set
+# CONFIG_AWS_SDK is not set
+# CONFIG_NETUTILS_CODECS is not set
+
+#
+# CURL Options
+#
+# CONFIG_ENABLE_CURL is not set
+# CONFIG_ERROR_REPORT is not set
+# CONFIG_ENABLE_IOTIVITY is not set
+# CONFIG_NETUTILS_JSON is not set
+# CONFIG_LIBTUV is not set
+# CONFIG_STRESS_TOOL is not set
+# CONFIG_VOICE_SOFTWARE_EPD is not set
+
+#
+# Application Configuration
+#
+
+#
+# Application entry point list
+#
+# CONFIG_ENTRY_MANUAL is not set
+CONFIG_ENTRY_HELLO=y
+# CONFIG_ENTRY_TESTCASE is not set
+CONFIG_USER_ENTRYPOINT="hello_main"
```

Comment:
This is tc config, but you set the entry point to hello_main. Is there any special reason?

Reply:
Fixed. thanks.